### ISSUE TITLE
v.1.5.3.1 and v 1.5.3.18a

### ISSUE THREAD

**Issue by blackvikingx** [w-okada/voice-changer#1302]
Which version is the most recent? v1.5.3.1 or 1.5.3.18a

**Comment by Kuuko-fokkusugaru** [comment]
> Which version is the most recent? v1.5.3.1 or 1.5.3.18a

The bigger the number, the newer the version. And, since 18 is bigger than 1, 18a is the latest version.

**Comment by blackvikingx** [comment]
So version 2 is the 18th, correct?

**Comment by Kuuko-fokkusugaru** [comment]
> So version 2 is the 18th, correct?

No, version 2 is the version that starts with the number two while version 1 starts with one. The v1.5.3.18a is the latest version of the v1 version of the software. 2.0.56 is the latest version of the alpha version 2 of the software.

**Comment by Kuuko-fokkusugaru** [comment]
Ti make it more simple to understand:

RVC1 latest version: v1.5.3.18a
RVC2 latest version (in development) v2.0.56 alpha
